sale_variant_configurator/models/sale_order.py

```diff
@@ -130,24 +128,3 @@ def product_uom_change(self):
             self.price_unit = self.env['account.tax']._fix_tax_included_price(
                 self._get_display_price(product_tmpl), product_tmpl.taxes_id,
                 self.tax_id)
-
-    @api.onchange('product_id')
-    def product_id_change(self):
-        if not self.product_id and self.product_tmpl_id:
-            vals = {}
-            name = self.product_tmpl_id.name_get()[0][1]
-            if self.product_tmpl_id.description_sale:
-                name += '\n' + self.product_tmpl_id.description_sale
-            vals['name'] = name
-            self.update(vals)
-        else:
-            return super(SaleOrderLine, self).product_id_change()
-
-    def _auto_init(self, context=None):
```

Comment:
Are you sure you can remove this? This is for avoiding to lose product_tmpl_id values each time you update the modules.